aikau/src/main/resources/alfresco/forms/controls/MultiSelect.js

```diff
@@ -159,8 +159,9 @@ define([
 
          /**
           * An object that defines the formats of the labels. See main module example for example.
+          * It should be a format string for each of the three label strings
           *
-          * @type {object} A format string for each of the three label strings
+          * @type {object}
```

Comment:
Damnit ... always struggle with this. The number of times I've fixed this myself in other places. Stupid JSDoc.